djangosaml2idp/migrations/0001_initial.py

```diff
@@ -42,4 +44,18 @@ class Migration(migrations.Migration):
             model_name='serviceprovider',
             index=models.Index(fields=['entity_id'], name='djangosaml2_entity__5fb9e3_idx'),
         ),
+        migrations.CreateModel(
+            name='PersistentId',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('persistent_id', models.CharField(max_length=256, verbose_name='User Persistent Id for this SP')),
```

Comment:
This migration does not look correct.

Reply:
Ah that's awkward, looks like I forgot to add in the new migration file. Fixed now.

Reply:
I've never used django migrations but what happens if you delete all of the migrations and then recreate them? My thought is that it would give you the create model migration as the model is now. Since this introduces a new model I don't think it should be 3 migrations. Especially since one creates a column and then modifies its type.

Reply:
At least personally, I tend to prefer granular migrations that can be more easily rolled back. Although on examination, it looks like I added the persistent id model by modifying the original migration for service providers; I'll undo that so that all the persistent id changes are contained in a second, separate migration,.

Reply:
Got it. Just seems like you could combine some of them, mainly just creating `persistent_id` as a UUID field instead of CharField and then modifying it, which I believe Django can do automagically out of the box. Like I said I don't use them enough to understand the proper way of doing it.

Reply:
Alright here we go, I agree this is a lot cleaner